library/misc/signal_channel/src/main/resources/quilt_signal_channel.accesswidener

```diff
@@ -0,0 +1,3 @@
+accessWidener v2 named
+
+transitive-accessible   class   net/minecraft/block/entity/BlockEntityType$BlockEntityFactory
```

Comment:
What is this used for? I dont see this class referenced anywhere in your code

Reply:
Implicitly converted to from a constructor reference in the testmod when creating a BlockEntityType with the builder. Had to be copied from the block_entity module due to transitive AWs not working

Reply:
Ah I see, alright

Is it possible to keep it in the testmod resources or does loom throw a fit with that?